packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -100,8 +100,6 @@ const showSpinner = computed<boolean>(
 const poller = new Poller();
 const pollResult = async (runId: string) => {
 	poller
-		.setInterval(5000)
-		.setThreshold(350)
```

Comment:
@Tom-Szendrey we should take this opportunity to adjust intervals and thresholds based on latest knowledge, if we expect the process to take a significant time to finish, specify nothing will give you a 2 minute window.
- ensemble-calibrate
- optimize

Reply:
I believe calibrate should be included in that list as well